### PR TITLE
#258: Fixed card image height issue

### PR DESCRIPTION
**Pull request by omkar-pd**
**Testing Steps:**

- Add any article with an image.
- Go to the homepage and check if the height of the new is the same as others.
- Refer to the following screenshot.
- Also check for the hover effects on image.

image

### CODE REVIEW

**Review by vbadkar — APPROVED** [review]
Working fine on mobile and tablet view as well on my side. 
image
Hover effect -
image

**Review by RChaubey16 — APPROVED** [review]
Working on my end too!
LGTM

Screenshot from 2022-02-01 15-08-56

**Review by Libbna — CHANGES_REQUESTED** [review]
image

On my end the issue is still there. See centre article's height. 
@omkar-pd

**Comment by omkar-pd** [comment]
@Libbna  check again

File: assets/scss/components/_card.scss [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+    object-fit: cover;
+    &:hover {
+      filter: brightness(0.6);
+    }
   }
 
   .card-body {
     .card-text {
-      min-height: 96px;
+      min-height: 110px;
```
Comment (Libbna) [inline assets/scss/components/_card.scss]: can you try to give min min-height: `120px;` ?  
I am still facing the issue.

**Review by Libbna — CHANGES_REQUESTED** [review]

File: assets/scss/components/_card.scss (hunk 1 above)
Comment (omkar-pd) [reply assets/scss/components/_card.scss]: Done

**Review by Libbna — APPROVED** [review]
image

Looks good.

**Pull request merged**